src/import.ml

```diff
@@ -86,3 +93,47 @@ let with_confirmation message ~yes ?(no = "Cancel") f =
   match choice with
   | Some true -> f ()
   | _ -> Promise.return ()
+
+(** Builds application-specific functionality around [Vscode.Position] *)
+module Position = struct
+  let compare p1 p2 =
+    let r = Position.compareTo p1 ~other:p2 in
+    if r < 0 then
+      Ordering.Less
+    else if r = 0 then
+      Equal
+    else
+      Greater
```

Comment:
I think `compare` should return an integer based on existing `compare` functions in the base library and this project.

It would also remove the need for `List.sort'`.

Reply:
I agree with your other comments, but let's keep this here: 
- it's better than dealings with ints
- it's the standard in ocamllsp; I'd hope we accept this idiom in this codebase as well